easybuild/main.py

```diff
@@ -272,6 +272,10 @@ def main(args=None, logfile=None, do_build=None, testing=False, modtool=None):
         search_easyconfigs(search_query, short=options.search_short, filename_only=options.search_filename,
                            terse=options.terse)
 
+    if options.update:
+        eb_file = find_easybuild_eb()
+        orig_paths.append(eb_file)
```

Comment:
I would try and make sure that *only* EasyBuild is installed, and throw an error if other easyconfigs are installed at the same time?